apps/src/code-studio/components/stageExtras/MazeThumbnail.jsx

```diff
@@ -5,7 +5,7 @@ import {getSubtypeForSkin} from '@cdo/apps/maze/mazeUtils';
 import MazeMap from '@cdo/apps/maze/mazeMap';
 import drawMap from '@cdo/apps/maze/drawMap';
 
-const assetUrl = path => '/blockly/' + path;
+const assetUrl = path => 'https://studio.code.org/blockly/' + path;
```

Comment:
Yuck, you're right.  Complete miss on my part when I opened this PR last night.  Sorry!

I know we set certain flags for the build environment.  Can I specify a new flag that only applies in [deploy-storybook.sh](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/script/deploy-storybook.sh)?

My concern is that the Storybook deploy only copies over existing built files, it doesn't actually run the build step.  Is there a lightweight way for me to detect when a story is running inside the deployed Storybook?